src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+  const dailyRent = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
+
+  if (days > 0 && days < 3) {
+    result = days * dailyRent;
+  }
+
+  if (days >= 3 && days < 7) {
+    result = days * dailyRent - threeDaysDiscount;
+  }
```

Comment:
```suggestion
  if (days >= 3 && days < 7) {
    return days * dailyRent - threeDaysDiscount;
  }
```
Do you really need the first check and result variable?
You can just immediately return the needed amount after checking for discounts.
So if nothing passed the requirements for discount, just return the default amount.